src/main/scala/com/typesafe/sbt/packager/docker/DockerPlugin.scala

```diff
@@ -2,6 +2,7 @@ package com.typesafe.sbt
 package packager
 package docker
 
+import java.io.File
```

Comment:
I think this is an issue with the weird `package` notation used in native-packager. At some point I will fix this. For now ignore this warning. Thanks for paying attention :)